lib/metasploit/framework/login_scanner/http.rb

```diff
@@ -315,18 +319,30 @@ def attempt_login(credential)
           Result.new(result_opts)
         end
 
+        protected
+
+        # Returns a boolean value indicating whether the request requires authentication or not.
+        #
+        # @param [Rex::Proto::Http::Response] response The response received from the HTTP endpoint
+        # @return [Boolean] True if the request required authentication; otherwise false.
+        def no_authentication_required?(response)
```

Comment:
can we invert the logic on this? make it `authentication_required?` the extra negative makes it harder for me to follow and it'll line up better with the comment

Reply:
logic reverted